### DISCUSSION TITLE
Postproc

### DISCUSSION THREAD

**Discussion by SebGue** [Eomys/pyleecan#273] [Q&A] [answered]
Hello, is there a tuto or other example how to use postproc classes? Is there a specifc place where to save postproc results?
(I want to calculate demagnetization and postproc may be suited quite well. Otherwise I think, I will add an OutMag method.)
BR Sebastian

**Comment by BonneelP** [comment, marked as answer]
Hello,

On our side we use the PostPlot classes to automatize the call to an existing plot command with the proper parameter to save the resulting image in the Results folder (there is a method in Output to get the path to the result folder and the user can overwrite the default path with a variable in the simulation object).
We then have the "Internal" objects that can store some data. But I think that having a dictionary in Output to store user-define output won't hurt :) OutPost object is meant to store some default parameter for the plot. 
So you could define a PostMagnetization class that would run the computation, save the results in output.UD_dict["mag"] and save a plot in the result folder for instance. Does it fit what you are looking for ? 

Best regards,
Pierre
